src/index.html

```diff
@@ -8,6 +8,107 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="stars--0">
+      <div class="star start">
+      </div>
+
+      <div class="star">
+      </div>
+
+      <div class="star">
+      </div>
+
+      <div class="star">
+      </div>
+
+      <div class="star">
+
+      </div>
+    </section>
+
+    <section class="stars--1">
+      <div class="star star_active start">
```

Comment:
You don't need any additional classes.
```suggestion
      <div class="stars__star">
```